src/launcher/features/apps/appsEffects.ts

```diff
@@ -160,35 +159,42 @@ const install =
     };
 
 export const installDownloadableApp =
-    (app: DownloadableApp, version?: string): AppThunk =>
+    (app: DownloadableApp, toVersion?: string): AppThunk =>
     dispatch => {
-        if (version == null) {
-            sendAppUsageData(EventAction.INSTALL_APP, app.source, app.name);
-        } else {
-            sendAppUsageData(
-                EventAction.INSTALL_APP_OLD_VERSION,
-                app.source,
-                `${app.name} (${version})`
-            );
-        }
+        sendAppUsageData(EventAction.APP_MANAGEMENT, {
+            action:
+                toVersion !== app.latestVersion
+                    ? 'Install Explicit Version'
+                    : 'Install',
+            ...app,
+            toVersion: toVersion ?? app.latestVersion,
+        });
 
         dispatch(installDownloadableAppStarted(app));
-        return dispatch(install(app, version));
+        return dispatch(install(app, toVersion));
```

Comment:
I don't immediately see the reason for the name change. Can you please elaborate?

I would suggest leaving it as `version` or call it `specificVersion` 😀